src/lib/app_registry.tsx

```diff
@@ -76,8 +76,12 @@ class Home extends RootContainer {
 class WorksForYou extends RootContainer {
   constructor(props) {
     super(props)
+
+    // This really has to be a boolean for Relay to understand it
+    const showSpecialNotification = props.selectedArtist && props.selectedArtist !== ""
```

Comment:
You can use the ol’ double-bang to ensure it’s a boolean:

```
$ node
> !!""
false
```

Which means you can probably make Relay happy like so:

```js
this.route = new Routes.WorksForYou({
  selectedArtist: props.selectedArtist,
  showSpecialNotification: !!props.selectedArtist,
})
```